metagpt/provider/azure_openai_api.py

```diff
@@ -43,8 +45,36 @@ def _make_client_kwargs(self) -> dict:
         return kwargs
 
     def _calc_usage(self, messages: list[dict], rsp: str) -> CompletionUsage:
-        return CompletionUsage(prompt_tokens=0, completion_tokens=0, total_tokens=0)
+        usage = CompletionUsage(prompt_tokens=0, completion_tokens=0, total_tokens=0)
+        if not self.config.calc_usage:
+            return usage
+
+        model_name = "gpt-35-turbo" if "gpt-3" in self.model.lower() else "gpt-4-turbo-preview"
```

Comment:
This judgment is risky, and users may take the name GPT35, or any other name

Reply:
Replaced by `pricing_plan` configuration